lib/util.js

```diff
@@ -162,7 +162,11 @@ export default class Util {
   }
 
   static isNFT(color) {
-    return color > 2 ** 15;
+    return (color > (2 ** 15)) && (color <= ((2 ** 15) + (2 ** 14)));
```

Comment:
@pinkiebell @johannbarbie why not just `(color > (2 ** 15))` here? Was there any reason? All NSTs are NFTs as well.